maps/mapsutil_test.go

```diff
@@ -227,3 +237,56 @@ func TestClear(t *testing.T) {
 		require.Empty(t, m2)
 	})
 }
+
+func TestFlatten(t *testing.T) {
+	t.Run("Flatten (flat-map)", func(t *testing.T) {
+		input := make(map[string]interface{})
+		expected := map[string]any{"item": 0, "item1": 1, "item2": 2}
+		for i, v := range expected {
+			input[i] = v
+		}
+		result := Flatten(input, ".")
+		require.EqualValues(t, expected, result)
```

Comment:
Maps iteration order is random, consider using `require.ElementsMatch` (ref: https://github.com/projectdiscovery/utils/issues/12) when comparing maps.

Reply:
Used `require.ElementsMatch`

Reply:
still using `require.EqualValues`